contracts/AccountBalance.sol

```diff
@@ -562,6 +563,14 @@ contract AccountBalance is IAccountBalance, BlockContext, ClearingHouseCallee, A
         return IIndexPrice(baseToken).getIndexPrice(twapInterval);
     }
 
+    function _isEnableMarkPrice(uint32 marketTwapInterval, uint32 premiumInterval) internal view returns (bool) {
```

Comment:
Because marketTwapInterval and premiumInterval are queried from external storage, if AccountBalance is upgraded before params are set, all transactions will be failed. 

Add sanity check function here to check all params required by mark price calculation are set.